Gruntfile.js

```diff
@@ -167,7 +167,7 @@ module.exports = function (grunt) {
       blanket_qunit: {
           all: {
               options: {
-                  urls: ["http://localhost:50047/results/js/index.html?coverage=true&lights=4"],
+                  urls: ["http://127.0.0.1:30047/results/js/index.html?coverage=true&lights=4"],
```

Comment:
Seems that we could pull host and port out and into variables instead of having the magic number 30047 in three places.